packages/web-app-admin-settings/src/mixins/general/resetLogo.ts

```diff
@@ -11,6 +11,9 @@ export default {
           label: () => {
             return this.$gettext('Reset logo')
           },
+          isEnabled: () => {
+            return this.$can('update', 'Logo')
```

Comment:
Shouldn't this be `delete`? 🤔

Reply:
I wanted to keep it close to the backend permissions (which is `change-logo` only). And technically, you update the logo by resetting it to default. But `delete` kinda applies as well... 🤷

Reply:
thought about it again... `delete` is actually too technical. Good that you kept it at `update`. 👍